pipelines/templates/tasks/upmpackages.yml

```
@@ -6,11 +6,18 @@ parameters:
   buildNumber: $(Build.BuildNumber)
   excludeBuildNumber: 0
 
+variables:
+- name: previewVersion
+  value: ""
+
 steps:
 - task: NodeTool@0
   inputs:
     versionSpec: '12.18.0'
-    
+
+- powershell: |
+  $previewVersion = ${{ parameters.buildNumber }}.Substring(4)
```

I think this needs to be explicitly stored in the yml variable:

```suggestion
    $previewVersion = ${{ parameters.buildNumber }}.Substring(4)
    echo "##vso[task.setvariable variable=previewVersion"]$previewVersion
```